time time
uint8 button
bool longPress
